app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/FullDetailsFragment.java

```diff
@@ -891,10 +892,21 @@ private void gotoSeries() {
         navigationRepository.getValue().navigate(Destinations.INSTANCE.itemDetails(UUIDSerializerKt.toUUID(mBaseItem.getSeriesId())));
     }
 
+    private void deleteItem() {
+        FullDetailsFragmentHelperKt.deleteItem(
+                this,
+                api.getValue(),
+                ModelCompat.asSdk(mBaseItem),
+                dataRefreshService.getValue(),
+                navigationRepository.getValue()
+        );
+    }
```

Comment:
The old implementation had a delete confirmation prompt which I found quite useful. Any plans to bring that back?

Reply:
I felt like I missed something, it was the dialog. Added it back now.

Reply:
LGTM